src/types.ts

```diff
@@ -122,6 +129,8 @@ export interface PipelineIntervention {
   accuracy: "high" | "medium" | "low" | "";
   is_alternative: boolean;
 
+  intervention_timescale: PipelineTimescale;
```

Comment:
Just an optional estimated completion year, not the full thing, based on the meeting yesterday?

Reply:
This is still the whole timescale sub-form -- I think we only want an optional completion year